sdks/python/src/opik/decorator/base_track_decorator.py

```diff
@@ -23,6 +23,7 @@
     inspect_helpers,
     span_creation_handler,
 )
+from .tracing_runtime_config import is_tracing_active
```

Comment:
please use `from . import tracing_runtime_config`
Opik's convention is to [import modules, not names](https://github.com/comet-ml/opik/tree/main/sdks/python#imports)